src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
```suggestion
export const BASE_URL = 'https://mate.academy/students-api/';
```
move it to the environment variables